collector/testdata/all_metrics.prom

```diff
@@ -71,19 +71,19 @@ snowflake_used_cloud_services_credits{service="mock_service_type",service_type="
 # TYPE snowflake_used_compute_credits gauge
 snowflake_used_compute_credits{service="another_mock_service_type",service_type="another_mock_service"} 0.25
 snowflake_used_compute_credits{service="mock_service_type",service_type="mock_service"} 0.5
-# HELP snowflake_warehouse_blocked_queries Average number of queries blocked by a transaction lock.
+# HELP snowflake_warehouse_blocked_queries Average load value for queries blocked by a transaction lock over the last 24 hours.
```

Comment:
I think there was an update to snowflake since this exporter was released, which included changes to a couple metric descriptions. The old file used for testing was outdated, so I updated the descriptions to match what is actually expected.

Reply:
Feel free to resolve